osdep/machines/glibc/common/include/features.h

```diff
@@ -3,7 +3,7 @@
 
 #if defined(_GLIBC2_12) || defined(_GLIBC2_14) || defined(_GLIBC2_15) \
 	|| defined(_GLIBC2_17) || defined(_GLIBC2_18) || defined(_GLIBC2_19) \
-	|| defined(_GLIBC2_31)
+	|| defined(_GLIBC2_27) || defined(_GLIBC2_31)
```

Comment:
i think there should be a matching `_GLIBC2_27` addition at the `#pragma TenDRA end` section at the bottom

Reply:
Ah yes! The closing scope isn't required, but it's a good idea to keep these hacks self-contained.

Reply:
Added in 217531e2b, thanks